crates/apub/src/objects/post.rs

```diff
@@ -40,11 +41,13 @@ use lemmy_db_schema::{
 };
 use lemmy_utils::{
   error::LemmyError,
-  utils::{check_slurs, convert_datetime, markdown_to_html, remove_slurs},
+  utils::{check_slurs_opt, convert_datetime, markdown_to_html, remove_slurs},
 };
 use std::ops::Deref;
 use url::Url;
 
+const MAX_TITLE_LENGTH: usize = 100;
```

Comment:
`name              | character varying(200)`

Reply:
I think 200 is way too long. Just tried with dev tools and a 200 char title stretches over 4 lines. So imagine Mastodon users forget about the title field, and just write a long paragraph in the beginning. It would make Lemmy almost unusable. Tbh i would even lower the maximum title length in our database.